bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
`StringIO` takes unicodes. I changed it to `BytesIO` for clarity (and thus avoiding the conversion).